docs/Examples/ErrorTypes.md

```diff
@@ -21,12 +21,12 @@ Or RxSwift way:
 }
 ```
 
-You can do that by a `switch` on different `cases` of `MoyaError`. In case of an `.Underlying` error you can also get the original `NSError` and its properties, e.g. `code` to be informed about `NSURLError` types like `NSURLErrorTimedOut` or `NSURLErrorNotConnectedToInternet`
+You can do that by a `switch` on different `cases` of `MoyaError`. In case of an `.underlying` error you can also get the original `NSError` and its properties. 
+
+e.g.) code to be informed about `NSURLError` types like `NSURLErrorTimedOut` or `NSURLErrorNotConnectedToInternet`
```

Comment:
Hm, not sure what you wanted to do here, but `e.g.)` doesn't seem like a desired result? 😄 Also I'd go with `code` instead of "code".